neo/test/utils/test_datasets.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+from neo.utils import download_dataset
+
+
+class TestDownloadDataset(unittest.TestCase):
+    def test_download_dataset(self):
+        local_path = download_dataset(
+            repo='https://gin.g-node.org/NeuralEnsemble/ephy_testing_data',
```

Comment:
I think this default url should only be defined in one place. Currently there's a 2nd and 3rd version in the `common_rawio_test.py` and `utils/datasets.py`

Reply:
Here it could be another url. We test the download only.